packages/react-native-renderer/src/ReactNativeAttributePayload.js

```diff
@@ -38,7 +38,7 @@ function defaultDiffer(prevProp: mixed, nextProp: mixed): boolean {
     return true;
   } else {
     // For objects and arrays, the default diffing algorithm is a deep compare
-    return deepDiffer(prevProp, nextProp);
+    return deepDiffer(prevProp, nextProp, {unsafelyIgnoreFunctions: true});
```

Comment:
This is creating a new object every time. Given that it's static, can we please hoist it out?